sdv/data_processing/data_processor.py

```diff
@@ -750,7 +753,10 @@ def fit(self, data):
         if constrained.empty:
             raise ValueError(
                 'The constrained fit dataframe is empty, synthesizer will not be fitted.')
-        LOGGER.info(f'Fitting HyperTransformer for table {self.table_name}')
+        table_name = self.table_name
+        if table_name is None:
+            table_name = ''
```

Comment:
actually since we're doing this everywhere, why don't we just handle it in the init? And do 
```python
self.table_name  = table_name or ''
```